CIPs/cip-0033.md

```diff
@@ -151,15 +151,17 @@ To be provided before reaching _review_ status.
 
 ## **Test Cases**
 
-To be provided before reaching _review_ status.
+[placeholder]
```

Comment:
```suggestion
The testing for cEUR should include transferring cEUR and being able to use the stability mechanism to mint and burn cEUR with CELO. A correct bucket update should be observed in the right block. It would be safe to test cUSD also to check this new deployment doesn't interfere with the already working implementation.
```